lib/pub_sub.rb

```diff
@@ -1,4 +1,5 @@
 require 'aws-sdk'
+require 'active_support/all'
```

Comment:
This isn't currently required in the gemspec.

I'd prefer it if we didn't add extra dependencies, but if we do they should be in the gemspec.